packages/base/matrix-room.gts

```diff
@@ -0,0 +1,415 @@
+import {
+  contains,
+  containsMany,
+  field,
+  Component,
+  Card,
+  primitive,
+  CardBase,
+  createFromSerialized,
+} from './card-api';
+import StringCard from './string';
+import DateTimeCard from './datetime';
+import IntegerCard from './integer';
+import MarkdownCard from './markdown';
+import { BoxelMessage } from '@cardstack/boxel-ui';
+import cssVar from '@cardstack/boxel-ui/helpers/css-var';
+import { formatRFC3339 } from 'date-fns';
+import Modifier from 'ember-modifier';
+import { type LooseSingleCardDocument } from '@cardstack/runtime-common';
+
+const roomMembers = new Map<string, RoomMemberCard>();
```

Comment:
This is more like `MatrixUsers` than room members, isn't it? There isn't anything room-specific about the mapping from user ID to member.

Reply:
there is actually--it's the room status which i hesitated in terms of putting on this card. but keep in mind that the whole reason we are receiving this event is because of our membership to a room. it's probably not a good idea to share this state outside of a room though, as that would leak across room authorization boundaries